tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -375,7 +670,14 @@ public static ExplorationTestDescription GetExplorationTestDescription(Explorati
                 IsGitSubmodulesRequired = true,
                 PathToUnitTestProject = "csharp/src/Google.Protobuf.Test",
                 SupportedFrameworks = new[] { TargetFramework.NETCOREAPP2_1 },
-                ShouldRun = false // Dictates that this exploration test should not take part in the CI
+                TestsToIgnore = new string[]
+                {
+                    "Google.Protobuf.CodedInputStreamTest.MaliciousRecursion",
+                    "Google.Protobuf.CodedInputStreamTest.MaliciousRecursion_UnknownFields",
+                    "Google.Protobuf.CodedInputStreamTest.RecursionLimitAppliedWhileSkippingGroup",
+                    "Google.Protobuf.JsonParserTest.MaliciousRecursion"
+                },
+                ShouldRun = true // Dictates that this exploration test should not take part in the CI
```

Comment:
This comment is wrong/doesn't match `ShouldRun`?

Reply:
Fixed in 6853dece4bb3cc91a684157e03ae7d3a18a3de0a